products/memgraph/upcoming/tutorials/marvel-universe.md

```diff
@@ -0,0 +1,165 @@
+One of the most flexible ways is to use regex matching (represented by the
+regex-matching operator "=~").
+
+```opencypher
+MATCH (h:Hero) WHERE
+h.name =~ ".*SPIDER.+"
+RETURN h.name as PotentialSpiderDude
+```
+
+We recommend you search for your heroes of interest this way, might save you
+some time!
+
+2) List all the heroes that have appeared together with Spider-Man/Peter Parker in a comic:
+
+```opencypher
+MATCH (:Hero {name: "SPIDER-MAN/PETER PARKER"})-[:AppearedInSameComic]->(h:Hero)
+RETURN DISTINCT h.name AS SpiderAssociate;
+```
+
+Wonder why we've used the "DISTINCT" keyword? Here's why - the data graph we got
+by loading the scraped MCU data (as used in the paper linked to in the
+introduction) contains an edge between two Hero nodes for every comic they
+appear in together. So, to remove duplicates, we had to plug in the "DISTINCT" keyword. 
+
+3) List all the comic issues where Spider-Man (Peter Parker) and Venom (Eddie Brock) appear together:
+
+```opencypher
+MATCH (:Hero {name: "SPIDER-MAN/PETER PARKER"})-[:AppearedIn]->(c:Comic)<-[:AppearedIn]-(:Hero {name: "VENOM/EDDIE BROCK"})
+RETURN c.name AS SpideyAndVenomComic;
+```
+
+4) List 10 heroes with whom Spider-Man (Peter Parker) appeared most frequently together:
+
+```opencypher
+MATCH (:Hero {name: "SPIDER-MAN/PETER PARKER"})-[r:AppearedInSameComic]->(h:Hero) WITH
+h.name AS SpideyBuddy,
+count(r) AS NumCollabs
+RETURN SpideyBuddy, NumCollabs
+ORDER BY NumCollabs DESC
+LIMIT 10;
+```
+
+5) List the 10 most popular heroes in the MCU:
+
+Quickly, name the five most popular heroes in the MCU! Alright, how did your
+brain decide what to give as the answer? We're assuming that you have no clue,
+but it vaguely has to do with the number and quality of connections each of
+those heroes have in your brain. However, how to explain the concept of "popular"
+to our database engine?
+
+Well, our philosophy is as follows - a popular hero is the one who's "known" by
+more other heroes, or in terms of our MCU graph, a hero that the other heroes have
+more connections (edges) to than some other hero is deemed "more popular".
+This philosophy is the one underlying Google's search engine, and the algorithm
+embodying it is PageRank, so it would be convenient if we could make use of it.
+
+However, the query engine doesn't support PageRank out-of-the-box, so we have to
+come up with a way to plug in PageRank to our database. That's precisely the
+purpose of [query modules](../reference_guide/query-modules.md)!
+
+Long story short, the query module system enables us to write C or Python modules
+that can access the data stored in our graph database, do some processing, and
+return the results of this processing to the query engine, so we can perform
+further queries on them. In this particular case, the PageRank algorithm is
+implemented as a Python module, and can be found in the query module directory
+"/usr/lib/memgraph/query_modules/", along with its description and the examples
```

Comment:
Put backticks instead of double quotes for the path (`/usr/lib/memgraph/query_modules`).

Reply:
Will do.